optimize.c

```diff
@@ -2164,7 +2164,7 @@ static void
 opt_init(opt_state_t *opt_state, struct icode *ic)
 {
 	bpf_u_int32 *p;
-	int i, n, max_stmts;
+	int i, n, max_stmts, block_memsize, edge_memsize;
```

Comment:
`block_memsize` and `edge_memsize` should probably be `size_t`, as 1) they shouldn't become negative and 2) on LP64 and LLP64 platforms, you want them to be bigger than 32 bits.

Reply:
thanks for your advice